t/python/t0005-kvs.py

```diff
@@ -311,7 +327,7 @@ def test_kvsdir_10_fill_initial_path(self):
             self.assertEqual(kd4["i.j.k"], "baz")
             self.assertEqual(kd4["i"]["j"]["k"], "baz")
 
-    def test_kvsdir_11_mkdir_initial_path(self):
+    def test_kvsdir_12_mkdir_initial_path(self):
```

Comment:
Still renumbering here?

Reply:
I decided to number within test "groups".  So its more limited than before.